core/services/gateway/config/config.go

```diff
@@ -24,10 +24,16 @@ type ConnectionManagerConfig struct {
 
 type DONConfig struct {
 	DonId         string
-	HandlerName   string
-	HandlerConfig json.RawMessage
+	HandlerName   string          // Deprecated: use Handler instead
+	HandlerConfig json.RawMessage // Deprecated: use Handler instead
 	Members       []NodeConfig
 	F             int
+	Handlers      []Handler
+}
+
+type Handler struct {
+	Name   string
```

Comment:
Alternatively, we could provide a list of method names here. This could simplify some code but complicate config so I'm not sure if it's worth it.